### ISSUE TITLE
PHP files which contain phtml in the name detect as a template

### ISSUE THREAD

**Issue by iGerchak** [magento/magento-coding-standard#147]
### Preconditions
1. Magento coding standard 4.0

### Steps to reproduce
1. Create Block with name AmpHtmlBlock.php with the following content:
```php
<?php

namespace Block;

/**
 * Class AmpHtmlBlock
 */
class AmpHtmlBlock
{
    /**
     *
     */
    public function test()
    {
        $this->test();
    }
}
```

### Expected result
There shouldn't be any warnings

### Actual result
```
FILE: .../Block/AmpHtmlBlock.php
-------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
-------------------------------------------------------------------------------------------
 15 | WARNING | Usage of $this in template files is deprecated.
-------------------------------------------------------------------------------------------
```

**Comment by ihor-sviziev** [comment]
@lenaorobei could you review this issue? As for me - we need to fix it

**Comment by lenaorobei** [comment]
@iGerchak thanks for reporting. @ihor-sviziev, sure, I agree it needs to be fixed.
